### ISSUE TITLE
@vue/prettier breaks this plugin

### ISSUE THREAD

**Issue by Maxim-Mazurok** [azeemba/eslint-plugin-json#71]
If I use this eslint config:
```
plugins: ["json-files"],
extends: ["@vue/prettier"],
```

Then this plugin doesn't work. It receives this JSON (added `console.log({json});` in `editStringJSON()`):
```
{
  json: 'module.exports = {\n' +
    '  "name": "web-watcher",\n' +
    '  "version": "0.2.0",\n' +
    '  "private": true,\n' +
    '  "scripts": {\n' +
```	

Is this something that I should report to `@vue/prettier` instead?
